app/components/progress-donut.ts

```diff
@@ -0,0 +1,43 @@
+import Component from '@glimmer/component';
+import { htmlSafe } from '@ember/template';
+import type { SafeString } from '@ember/template/-private/handlebars';
+
+interface Signature {
+  Element: SVGElement;
+
+  Args: {
+    total: number;
+    completed: number | undefined;
```

Comment:
```suggestion
    completed: number;
```

(Let's force callers to send zero explicitly if they need to)